Lib/test/test_clinic.py

```diff
@@ -13,25 +13,19 @@
 import os.path
 import re
 import sys
-import types
 import unittest
 
 test_tools.skip_if_missing('clinic')
 with test_tools.imports_under_tool('clinic'):
     import clinic
     from clinic import DSLParser
 
-
-def default_namespace():
-    ns = types.SimpleNamespace()
-    ns.force = False
-    ns.limited_capi = clinic.DEFAULT_LIMITED_CAPI
-    return ns
+DEFAULT_LIMITED_C_API = False
```

Comment:
I added it to clinic.py to maybe change the default later. I don't think that it's relevant to have such constant in test_clinic. Just write False?

Reply:
Hmm, yes, maybe this is over-engineered right now :)